crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -420,7 +420,7 @@ define_tedge_config! {
 
         /// Set of MQTT topics the Azure IoT mapper should subscribe to
         #[tedge_config(example = "te/+/+/+/+/a/+,te/+/+/+/+/m/+,te/+/+/+/+/e/+")]
-        #[tedge_config(default(value = "te/+/+/+/+/m/+,te/+/+/+/+/e/+,te/+/+/+/+/a/+,tedge/health/+,tedge/health/+/+"))]
+        #[tedge_config(default(value = "te/+/+/+/+/m/+,te/+/+/+/+/e/+,te/+/+/+/+/a/+,te/+/+/+/+/status/health"))]
```

Comment:
```suggestion
        #[tedge_config(default(value = "te/+/+/+/+/m/+,te/+/+/+/+/e/+,te/+/+/+/+/a/+,te/+/+/+/+/cmd/health/check"))]
```
Surprised to see that these mappers have not subscribed to the `cmd/health/check` topic. How will they respond to health check requests from the watchdog without that?

Reply:
These are subscriptions the user can opt-out, say to publish only measurements. In any case, the mappers are subscribing to the topics which are required to well behave as `cmd/health/check`.

Reply:
But how it will work if the mapper has to be frequently checked for its health by a `watchdog`

Reply:
The `HealthMonitorBuilder`, used by all the tedge components, is responsible for subscribing to those `health/check` messages, used by the watchdog. You can't/shouldn't opt out of it, which is why it is not exposed via this config parameter.